### DISCUSSION TITLE
Config template with Jinja include function

### DISCUSSION THREAD

**Discussion by fodil1** [netbox-community/netbox#14348] [General]
Hi guys,

Do you know if the netbox config template supports the include feature by any chance?

{% include 'users.j2' %}

**Comment by gdprdatasubect** [comment]
Think so…

https://github.com/jeremystretch/example-config-templates/blob/master/roles/access-switch.j2

**Reply by fodil1** [reply]
Working great!

Thank you

**Comment by whtkam** [comment]
If I create two configuration templates in Netbox, is it possible to combine these two using {% include 'xxxx.j2' %}. What will be the right path or reference?
Thanks

**Comment by fodil1** [comment]
Hey,

I used this in my templates, the files are located at the same level in the repo.

***
{#
    FortiAnalyzer Configuration
#}
{%- include 'Fortigate_FAZ_Configlet.j2' %}
{#
    Tacacs Configuration
#}
{%- include 'Fortigate_Tacacs_Configlet.j2' %}
***

**Comment by whtkam** [comment]
Thanks – it must be the full path to the root of the repo! - now it works!
